masonry/src/contexts.rs

```diff
@@ -333,6 +326,7 @@ impl_context_method!(
 // --- MARK: CURSOR ---
 // Cursor-related impls.
 impl_context_method!(EventCtx<'_>, {
+    // TODO - Rewrite doc
```

Comment:
Not sure if there's something missing yet in the code, but it would help reviewing this IMO.

Reply:
I'm leaving this for a later PR.